src/main/java/paymelah/logic/parser/ParserUtil.java

```diff
@@ -121,4 +123,38 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         }
         return tagSet;
     }
+
+    /**
+     * Parses a {@code String description} into a {@code Description}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @param description The description to turn into a {@code Description} object.
+     * @return The corresponding {@code Description}.
+     * @throws ParseException if the given {@code description} is invalid.
+     */
+    public static Description parseDescription(String description) throws ParseException {
+        requireNonNull(description);
+        String trimmedDescription = description.trim();
```

Comment:
Since we are trimming the description, it is no longer possible for the first character of description to be a whitespace character. Should we change the Description validation regex to reflect this?

Reply:
Other regexes like Address also have the "no whitespace" explicitly included. I agree it's not very necessary, but maybe for testing it is better to ensure leading white spaces are caught regardless?

Reply:
That makes sense, I agree.